openedx/openedx-platform
# [ENT-628] Add `ENTERPRISE_TAGLINE`, `PLATFORM_DESCRIPTION`, and edx-enterprise version bump.

The changes in this PR are all linked together:

1. Adds the custom Enterprise tagline to the top of the the Enterprise pages.
1. Add the `PLATFORM_DESCRIPTION` configuration variable for describing the platform; for now, to be used on the Enterprise landing pages.
1. Version bump edx-enterprise to use the most recent version of the programs landing page (WIP).

The corresponding configuration PR is https://github.com/edx/configuration/pull/4083

**JIRA tickets**: [ENT-628](https://openedx.atlassian.net/browse/ENT-628)

**Screenshots**:

Tagline:

![selection_003](https://user-images.githubusercontent.com/10018065/30269901-13576f10-9704-11e7-89e6-13cb7ffffd5d.png)

Platform description (wip):

"blah"

**Testing instructions**:

This requires [ENT-628](https://openedx.atlassian.net/browse/ENT-628) to have a PR up first, which it will soon. When there is a PR up for it:

1. Use that PR's version of `edx-enterprise` in your `edx-platform` devstack.
1. Go to the programs enrollment landing page. Notice the tagline you entered being in the header next to the logo. Notice also the platform description on the sidebar.
1. Go to the data sharing consent page. Notice the tagline you entered being in the header next to the logo.
1. Go to the course enrollment landing page. Notice the tagline you entered being in the header next to the logo.

**Reviewers**

**Settings**
```yaml
EDXAPP_FEATURES:
  ENABLE_COMBINED_LOGIN_REGISTRATION: true
  ENTERPRISE_TAGLINE: "Why, hello there."
  PLATFORM_DESCRIPTION: "What a fancy description!"
```
